unit-tests-jest/numbers.test.js

```diff
@@ -0,0 +1,113 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+describe('evenNumbers(numbers)', () => {
+
+  it('returns a new array that contains only the even numbers in the given array', () => {
+    const numbers = [1, 4, 5, 10, 0];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([4, 10, 0]);
+  });
+
+  it('returns negative numbers', () => {
+    const numbers = [-1, -2, 0];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([-2, 0]);
+  });
+
+  it('doesn`t return infinite numbers', () => {
+    const numbers = [-1, -2, 0, Infinity, -Infinity];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([-2, 0]);
+  });
+
+  it('doesn`t return NaN', () => {
+    const numbers = [NaN];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([]);
+  });
+
+  it('doesn`t return non-integers', () => {
+    const numbers = [2, 4, 1.1, 2.2];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([2, 4]);
+  });
+
```

Comment:
Very thorough set of tests, several of which I missed in my own solution. Well done!